packages/salesforcedx-vscode-core/package.json

```diff
@@ -551,6 +551,10 @@
         {
           "command": "sfdx.force.internal.refreshsobjects",
           "when": "sfdx:project_opened"
+        },
+        {
+          "command": "sfdx.force.launch.replay.debugger",
+          "when": "sfdx:project_opened && editorLangId == 'apex'"
```

Comment:
This applies to apex files, and this command is available for both *.cls files and *.apex files.

Reply:
This is outdated, and it is now:
`"when": "sfdx:project_opened && resource =~ /.\\.(cls|apex|log)?$/"`